utils/node_local_test.py

```diff
@@ -1521,8 +1521,8 @@ def il_cmd(self, cmd, check_read=True, check_write=True, check_fstat=True):
         print(f'Logged il to {log_name}')
         print(ret)
 
-        if self.caching:
-            check_fstat = False
+        check_fstat = check_fstat and not self.caching and \
+            look_for_library_call(self.conf, cmd, '__fxstat')
```

Comment:
This would be called for every il_cmd invocation and there are probably dozens so it could/should be saved in `conf`

Reply:
I thought about that and wanted to not assume different executables ended up using the same libraries. And, I don't think it's going to affect overall runtime to just do this every time. Do you feel differently?